app/src/main/res/drawable-night/ic_content_copy.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Remove these, they are not needed anymore

Reply:
Done